Add sections on security levels and tiering (#153)
sa-tre/satre-specification: docs/source/pillars/information_governance.md

* - You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data
  - This may include multi-factor authentication (MFA), ID checks or email/phone verification
  - Mandatory

> * - You have a clear method of communicating the level of security controls present in the TRE for all roles within your TRE organisation

Reviewer: ```suggestion
* - You must clearly communicate the security controls of your TRE to all roles.
```

I've tried to tidy that but I'm not exactly sure what it means or how it relates to other additions in this PR.

Again, I'm not sure if this is really about the controls.

Reviewer 2: I think this can be incorporated with the above about communicating what your TRE can/can't do

Reviewer: OK, feel like the consensus is to remove this?

Reviewer: Updated in https://github.com/sa-tre/satre-specification/commit/eefc4621c858d9c429ceac51f1f52963563b6528

>   - This will be particularly important researchers working with the data, to understand what actions they are and are not permitted to take.

Reviewer: ```suggestion
  - People in every role should understand the security controls, the need for those controls, and how those controls impact on their responsibilities.
    For example, it should be clear what actions people are not permitted to take when controls are in place to prevent them.
```

Is this what we mean?

Reviewer 3: I think users knowing which actions are and aren't allowed is part of user training. It isn't inherent to the idea of security tiers. As above, I'm not sure we need this point at all.

Reviewer: Agreed.

There isn't an item in "Training management and delivery" which specifically says allowed/not allowed actions should be covered.
Do we want to add this there or not include it at all.

Reviewer 3: I think we should add one (can be in this PR) that says something like "You must ensure that users are aware which actions are and are not permitted in your TRE // This is likely to depend on the security controls in place in your TRE // Mandatory"

Reviewer 2: Would this user point not be included in a user terms of use point in information governance?
